regression/goto-cc-goto-analyzer/instrument_preconditions_locations/test.desc

```diff
@@ -0,0 +1,15 @@
+CORE
+test.c
+--verify
+Checking assertions
+^EXIT=0$
```

Comment:
Could you check for at least one of the assertions?

Reply:
Tested for the first verification  result.